samples/client/petstore/scala-sttp/src/main/scala/org/openapitools/client/api/PetApi.scala

```diff
@@ -67,7 +67,7 @@ class PetApi(baseUrl: String) {
   def findPetsByStatus(status: Seq[String]
 ): Request[Either[ResponseError[Exception], Seq[Pet]], Nothing] =
     basicRequest
-      .method(Method.GET, uri"$baseUrl/pet/findByStatus?status=$status")
+      .method(Method.GET, uri"$baseUrl/pet/findByStatus?status=${ status }")
```

Comment:
Looks promising! Does it make sense to wrap only those which have backticks or it is not easy?

Reply:
It it for sure an additional complication which doesn't bring any value IMO.